src/devices/sportstechbike/sportstechbike.cpp

```diff
@@ -99,12 +99,7 @@ void sportstechbike::update() {
 
         QSettings settings;
         uint8_t noOpData[] = {0xf2, 0xc3, 0x07, 0x04, 0x00, 0x00, 0x00, 0x00, 0xff, 0xff, 0xbe};
-        if (requestResistance < 0) {
-            requestResistance = 0;
-        }
-        if (requestResistance > 23) {
-            requestResistance = 23;
```

Comment:
Unusual, the max_resistance constant from the header is 24.

Reply:
@drmason789 because the resistance levels are 24 but the bike wants it starting from 0. we can replace that 23 with max_resistance - 1